controllers/front/OrderDetailController.php

```diff
@@ -209,6 +213,11 @@ public function getBreadcrumbLinks()
             'url' => $this->context->link->getPageLink('history'),
         );
 
+        $breadcrumb['links'][] = array(
+            'title' => $this->reference,
```

Comment:
I think you must check if reference is not empty

Reply:
Added